can/tests/test_dispatch.cpp

```diff
@@ -35,7 +36,13 @@ SCENARIO("Dispatcher") {
         auto l2 = Listener{};
         auto buff = BufferType{1};
         uint32_t arb_id = 1234;
-        auto subject = Dispatcher(l1, l2);
+        struct CheckForNodeId {
+            NodeId node_id;
+            auto operator()(uint32_t arbitration_id) const { return true; }
```

Comment:
Since this just returns true, can we use a lambda in the `Dispatcher` declaration that simply returns true?